src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -280,6 +287,10 @@ bool CountryInstance::add_reform(Reform const& new_reform) {
 	}
 }
 
+bool CountryInstance::set_value_for_slider(SliderManager& slider, int const* new_value) {
```

Comment:
As mentioned previously there's no need for a `SliderManager`, and this function should probably take a `Strata const&` and an `slider_value_t::int_type` as arguments - then you can use the strata to look up the right slider in the previously-mentioned `IndexedMap<Strata, slider_value_t>` and set it's value to the other argument.
We only need an external setter function for the sliders' values, all the min/max setting will be done internally in the country gamestate update function based off of modifier effect totals